program_info/org.prismlauncher.PrismLauncher.desktop.in

```diff
@@ -8,6 +8,6 @@ Exec=@Launcher_APP_BINARY_NAME@
 StartupNotify=true
 Icon=org.prismlauncher.PrismLauncher
 Categories=Game;ActionGame;AdventureGame;Simulation;
-Keywords=game;minecraft;launcher;mc;multimc;polymc;
+Keywords=game;minecraft;launcher;mc;multimc;prism;
```

Comment:
both `prism` and `launcher` can go, since [le spec](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#recognized-keys) says they shouldn't be redundant with `Name`. (i love specs)

also I don't mind removing `multimc` too.